manifest.php

```diff
@@ -33,11 +33,11 @@
     'label' => 'QTI Portable Custom Interaction',
     'description' => '',
     'license' => 'GPL-2.0',
-    'version' => '4.0.0',
+    'version' => '4.0.1',
     'author' => 'Open Assessment Technologies SA',
     'requires' => array(
         'tao' => '>=9.2.2',
-        'taoQtiItem' => '>=11.0.0'
+        'taoQtiItem' => '>=11.2.0'
```

Comment:
I think that is not really good practice - while you are increasing only `patch` version  on extension together with `minor` on dependency you may break a lot of envs, where all version in `composer.json` are fixed to download only patches, like `~4.0.0`